common/src/main/java/com/revenuecat/purchases/common/ProductInfo.kt

```diff
@@ -1,18 +1,18 @@
 package com.revenuecat.purchases.common
 
-import com.android.billingclient.api.SkuDetails
+import com.revenuecat.purchases.models.ProductDetails
 
 class ProductInfo(
     val productID: String,
     val offeringIdentifier: String? = null,
-    val skuDetails: SkuDetails? = null
+    val productDetails: ProductDetails? = null
 ) {
 
-    val price: Double? = skuDetails?.priceAmount
-    val currency: String? = skuDetails?.priceCurrencyCode
-    val duration: String? = skuDetails?.subscriptionPeriod?.takeUnless { it.isEmpty() }
-    val introDuration: String? = skuDetails?.introductoryPricePeriod?.takeUnless { it.isEmpty() }
-    val trialDuration: String? = skuDetails?.freeTrialPeriod?.takeUnless { it.isEmpty() }
+    val price: Double? = productDetails?.priceAmountMicros?.div(1000000.0)
```

Comment:
Yes, it is. `ProductDetails` has everything that `SkuDetails` has and some of it is nullable for Amazon. Since I could calculate `priceAmountMicros` based on the Amazon price, I figured I should just add it. We can alternatively do two things:

1. Have it as null in Amazon, which I don't like, because I hate nulls
1. Remove it from `ProductDetails`, if someone wants to get it, they can get the SkuDetails using the originalJSON property. I like this, but then, should we do the same for things like `introPrice`? Although, I really don't think anyone is accessing `priceAmountMicros`

Reply:
I noticed I was confused here between `ProductInfo` and `ProductDetails`. `ProductDetails` already has a price that's a string that contains the currency code. We could add a priceDouble?